applications/nrf_desktop/README.rst

```diff
@@ -2012,18 +2012,29 @@ Background Device Firmware Upgrade
 ==================================
 
 The nRF Desktop application uses the :ref:`nrf_desktop_config_channel` and :ref:`nrf_desktop_dfu` for the background DFU process.
-From the application perspective, the update image transfer during the background DFU process is very similar for both MCUboot and B0 bootloader.
+From the application perspective, the update image transfer during the background DFU process is very similar for all supported configurations:
+
+* MCUboot
+* Secure Bootloader
+
 The firmware update process has three stages, discussed below.
 At the end of these three stages, the nRF Desktop application will be rebooted with the new firmware package installed.
 
 .. note::
   The background firmware upgrade can also be performed over the Simple Management Protocol (SMP).
-  For more details about the DFU over SMP, read the :ref:`nrf_desktop_ble_smp` or :ref:`nrf_desktop_dfu_mcumgr` documentation.
+  For more details about the DFU procedure over SMP, read the documentation of relevant modules:
+
+  * :ref:`nrf_desktop_ble_smp` (MCUboot)
+  * :ref:`nrf_desktop_dfu_mcumgr` (MCUboot)
+    The module uses the :ref:`nrf_desktop_dfu_lock` to synchronize non-volatile memory access with other DFU methods.
```

Comment:
`Because of that this module should be used for configurations that enable multiple DFU transports (e.g. if a configuration also enables :ref:`nrf_desktop_dfu`)`.